pkg/util/etcd/etcd.go

```diff
@@ -7,7 +7,8 @@ import (
 )
 
 const (
-	BackupStorageContainerBase = "kubernikus-etcd-backup"
+	BackupStorageContainerBase               = "kubernikus-etcd-backup"
+	BackupStorageContainerMinimumFreeStorage = 500000000 // 500MiB
```

Comment:
I guess this will break all our soak test as we only have 100Mib of quota there if I'm not mistaken

Reply:
Yes, this is true. We can change this to 100MiB but this won't be enough in most cases.